code/modules/integrated_electronics/core/integrated_circuit.dm

```diff
@@ -10,16 +10,17 @@ a creative player the means to solve many problems.  Circuits are held inside an
 
 // This should be used when someone is examining while the case is opened.
 /obj/item/integrated_circuit/proc/internal_examine(mob/user)
-	to_chat(user, "This board has [inputs.len] input pin\s, [outputs.len] output pin\s and [activators.len] activation pin\s.")
+	. = list()
+	. += "This board has [inputs.len] input pin\s, [outputs.len] output pin\s and [activators.len] activation pin\s."
 	for(var/datum/integrated_io/I in inputs)
 		if(I.linked.len)
-			to_chat(user, "The '[I]' is connected to [I.get_linked_to_desc()].")
+			. += "The '[I]' is connected to [I.get_linked_to_desc()]."
 	for(var/datum/integrated_io/O in outputs)
 		if(O.linked.len)
-			to_chat(user, "The '[O]' is connected to [O.get_linked_to_desc()].")
+			. += "The '[O]' is connected to [O.get_linked_to_desc()]."
 	for(var/datum/integrated_io/activate/A in activators)
 		if(A.linked.len)
-			to_chat(user, "The '[A]' is connected to [A.get_linked_to_desc()].")
+			. += "The '[A]' is connected to [A.get_linked_to_desc()]."
 	any_examine(user)
 	interact(user)
```

Comment:
At a quick glance it does not seem like internal_examine is being called anywhere?

Reply:
apparently not?